cmd/server/entrypoint.go

```diff
@@ -73,17 +73,15 @@ func StartProxySvr(cliCtx *cli.Context) error {
 }
 
 // TODO: we should probably just change EdaClientConfig struct definition in eigenda-client
-// to have a `json:"-"` tag on the SignerPrivateKeyHex field, to prevent the privateKey from being marshaled at all
 func prettyPrintConfig(cliCtx *cli.Context, log log.Logger) error {
 	// we read a new config which we modify to hide private info in order to log the rest
 	cfg := server.ReadCLIConfig(cliCtx)
-	cfg.EigenDAConfig.EdaClientConfig.SignerPrivateKeyHex = "HIDDEN"
-	cfg.EigenDAConfig.VerifierConfig.RPCURL = "HIDDEN"
-
 	configJSON, err := json.MarshalIndent(cfg, "", "  ")
 	if err != nil {
 		return fmt.Errorf("failed to marshal config: %w", err)
 	}
+	cfg.EigenDAConfig.EdaClientConfig.SignerPrivateKeyHex = "" // marshaling defined in client config
```

Comment:
Why did you move this after the marshaling? Like this it doesn’t do anything

Reply:
it does actually - the `SignerPrivateKey` is stored as part of the `EigenDAClientConfig` which is defined within layr-labs/eigenda

Reply:
The log line below is
```
log.Info(fmt.Sprintf("Initializing EigenDA proxy server with config: %v", string(configJSON)))
```
which prints the already marshalled config. So changing the fields of cfg here above doesn't do anything.

Reply:
🤦‍♂️

Reply:
[53421d7](https://github.com/Layr-Labs/eigenda-proxy/pull/194/commits/53421d71c3e4c96761395078089c3e3c0411a401)